src/main/java/seedu/duke/helper/command/ViewAppointmentCommand.java

```diff
@@ -0,0 +1,17 @@
+package seedu.duke.helper.command;
+
+import seedu.duke.assets.List;
+import seedu.duke.exception.NotFoundException;
+import seedu.duke.status.Status;
+
+public class ViewAppointmentCommand extends Command {
+    public ViewAppointmentCommand(String[] parameterArray) {
+        super(parameterArray);
+    }
+
+    public Status execute(List appointmentList) throws NotFoundException {
+        String parameters = String.join(",",parameterArray);
```

Comment:
Will this work if parameterArray is null?

Reply:
It shouldn't be null because in Parser there is 
`    public static Command parseViewAppointment(String parameters) throws HalpmiException {
        String[] viewAppointmentParameters = minParameterCheck(parameters, 2);
        return new ViewAppointmentCommand(viewAppointmentParameters);
    }`
which verifys that it should be length 2